.editorconfig

```diff
@@ -0,0 +1,11 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+
+[*.{fs,fsi,fsx}]
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
```

Comment:
it's supposed to be commited, it's for many many editors, 

![](http://i.imgur.com/sWmMSam.png)